micro-ui/web/micro-ui-internals/packages/modules/pgr/src/pages/employee/new-inbox.js

```
@@ -0,0 +1,67 @@
+import React, { useMemo,useState,useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { Header, InboxSearchComposer , Loader } from "@egovernments/digit-ui-react-components";
+import inboxConfigPGR from "../inboxConfigPGR"
+import { useLocation } from "react-router-dom";
+const Inbox_v2 = () => {
+  const { t } = useTranslation()
+  //fetch this config from mdms and pass it to the preProcess fn
+  const location = useLocation()
+   let configs = inboxConfigPGR();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'configs' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)